bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableConfiguration.java

```diff
@@ -184,20 +185,21 @@ public Configuration toHBaseConfig() {
   }
 
   /**
-   * Creates a new Builder from the configuration
-   * @return A new {@link Builder}
+   * Creates a new {@link Builder} object from the configuration.
+   * @return A new {@link Builder}.
```

Comment:
maybe: "@returns A new {@link Builder} object containing the existing configuration."?